File: 2026-07-20-countblinks.cs
using System.Text.RegularExpressions;

static int countBlinks(string input)
{
    var matches = Regex.Matches(input, "_.*?_");
    if (matches.Count > 0)
    {
        return matches.OrderByDescending(i => i.Length).First().Length - 2;
    }
    return 0;
}

var inputs = new List<string>
{
    "_..._",
    "_._.._...._",
    "....._",
    "._._.."
};
System.Console.WriteLine(countBlinks(inputs[0]));
System.Console.WriteLine(countBlinks(inputs[1]));
System.Console.WriteLine(countBlinks(inputs[2]));
System.Console.WriteLine(countBlinks(inputs[3]));

File: 2026-07-20-countblinks.js
function countBlinks(input) {
    const re = /_.*?_/g
    const arr = input.match(re)
    if (arr !== null) {
        return arr.reduce((a, b) => (a.length > b.length ? a : b)).length - 2
    }
    return 0
}

const inputs = ['_..._', '_._.._...._', '....._', '._._..']
console.log(countBlinks(inputs[0]))
console.log(countBlinks(inputs[1]))
console.log(countBlinks(inputs[2]))
console.log(countBlinks(inputs[3]))
